assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -106,15 +110,56 @@ class PageEdit extends Component {
 
 			mediaType = media.type;
 		}
+		const mediaUrl = has( media, [ 'sizes', MAX_IMAGE_SIZE_SLUG, 'url' ] ) ? media.sizes[ MAX_IMAGE_SIZE_SLUG ].url : media.url;
+
+		let isExcessiveVideoSize = false;
+		if ( VIDEO_BACKGROUND_TYPE === mediaType ) {
+			isExcessiveVideoSize = this.isVideoSizeExcessive( media.filesizeInBytes, media.fileLength );
```

Comment:
Minor point, but this could be simplified to:

```js
const isExcessiveVideoSize = VIDEO_BACKGROUND_TYPE === mediaType && this.isVideoSizeExcessive( media.filesizeInBytes, media.fileLength );
```

Reply:
Good idea, https://github.com/ampproject/amp-wp/pull/2509/commits/d2f9ea1c040f24585fe582858cc8f4c4068341bf does that.